packages/platform-sdk-ark/src/services/ledger.ts

```diff
@@ -110,14 +110,24 @@ export class LedgerService implements Contracts.LedgerService {
 				hasMore = collection.isNotEmpty();
 			} else {
 				const path = `m/44'/${slip44}'/0'`;
+				let initialAddressIndex = 0;
+
+				if (options?.startPath) {
+					/*
+					 * Get the address index from expected format `m/purpose'/coinType'/account'/change/addressIndex`
+					 */
+					const startAddressIndex = options.startPath.split("/")[5];
```

Comment:
Use https://github.com/ArkEcosystem/platform-sdk/blob/master/packages/platform-sdk-crypto/src/bip44.ts#L64-L89 to parse the path.